docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -425,7 +427,8 @@ For more information, see link:https://quarkus.io/blog/path-resolution-in-quarku
 [[map-security-identity-roles]]
 === Map `SecurityIdentity` roles
 
-Winning role-based policy can map the `SecurityIdentity` roles to the deployment-specific roles.
+When a security policy's `paths` property contains the most specific path that matches the current request path, it takes precedence over other security policies with matching paths and is said to win.
```

Comment:
@rolfedh We have the note above, thanks for adding it, can we try to avoid duplicating it as it reads unusual  ? We have the definition of the winning policy.

May be we can say `The winning role-based policy that was chosen to authorize the current request can map `SecurityIdentity` roles to deployment-specific roles.`